ncm-cups/src/main/pan/components/cups/schema.pan

```diff
@@ -1,8 +1,4 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-
-declaration template components/cups/schema;
+${componentschema}
```

Comment:
you have to remove the bind statement at the end (it's in config), and rename the main component type to `${project.artifactId}_component`
if no unittests fail without these modifications, you have to modify an existing one so the new component config is actually used (see http://www.quattor.org/documentation/2017/02/10/components-spring17-cleaning.html#add-a-unittest)